client/src/App.js

```diff
@@ -1,20 +1,28 @@
-import React from "react";
-import { MuiThemeProvider } from "@material-ui/core";
+import React from 'react';
+import { Container } from '@material-ui/core';
+import { ThemeProvider } from '@material-ui/core/styles';
+import { landingTheme, dialogTheme } from "./themes/theme";
 import { BrowserRouter, Route } from "react-router-dom";
-
-import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
-
-import "./App.css";
+import LoginDialog from './pages/LoginDialog'
+import SignUpDialog from './pages/SignUpDialog'
 
 function App() {
-  return (
-    <MuiThemeProvider theme={theme}>
-      <BrowserRouter>
-        <Route path="/" component={LandingPage} />
-      </BrowserRouter>
-    </MuiThemeProvider>
-  );
+	return (
+		<Container>
+			<ThemeProvider theme={landingTheme}>
+				<BrowserRouter>
+					<Route path="/" component={LandingPage} />
+				</BrowserRouter>
+			</ThemeProvider>
+			<ThemeProvider theme={dialogTheme}>
+				<BrowserRouter>
+					<Route path="/login" component={LoginDialog} />
+					<Route path="/sign-up" component={SignUpDialog} />
+				</BrowserRouter>
```

Comment:
As Aecio was saying earlier, we probably want to be using one universal theme for our app that gets applied to all components. We only use one background color for instance, so having the background color in the theme, as well as font family, font size, primary color, secondary color, etc.

Reply:
Yes, and the alternatives for styling the dialogs without themes were:
1) Create a single style with makeStyles and import it and use it on both Signin and Signup components
2) Create a CustomDialog component that uses {children} props, wrap the Sgnin and Signuo components with it. An example from a previous cohort:
 - Reusable component: https://github.com/hatchways/team-ginger/blob/dev/client/src/components/AccountForm.js
- Usage: https://github.com/hatchways/team-ginger/blob/dev/client/src/components/SignupForm.js